jrnl/exception.py

```diff
@@ -30,7 +30,13 @@ def _get_error_message(self, **kwargs):
 
                 Removing this file will allow jrnl to save its configuration.
             """
-            )
+            ),
+            "LineWrapTooSmallForDateFormat": textwrap.dedent(
+                """
+                The provided linewrap value of {config_linewrap} is too small
+                to display the timestamps in the configured time format.
+                """
+            ),
```

Comment:
I really like the use of this custom exception. A couple thoughts:

* Can we tell the user what the minimum allowable value is in this format?
* As with the other exception messages, we should also tell the user what they can do to fix it. Something like: "Modify your linewrap configuration or use --config-override to increase it."